webpack.config.js

```diff
@@ -2,6 +2,8 @@ const webpack = require('webpack')
 const path = require('path')
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
 const PROD = (process.env.NODE_ENV === 'production')
+const pluginDirectory = path.basename(__dirname)
+const chunkPath = `/wp-content/plugins/${pluginDirectory}/dist/assets/js/`
```

Comment:
We need to be able to get rid of `/wp-content/plugins/` in the config.

Reply:
I think that's the simplest way possible. Here is another way.

const pluginDirectory = path.basename(__dirname)
const wpPath = path.basename(path.resolve('../../'))
const plgPath = path.basename(path.resolve('../'))
const publicPath = `/${wpPath}/${plgPath}/${pluginDirectory}/dist/assets/js/`